proto/xudrpc.proto

```diff
@@ -422,6 +422,8 @@ message CreateNodeResponse {
   repeated string seed_mnemonic = 1;
   // The list of lnd clients that were initialized.
   repeated string initialized_lnds = 2;
+  // The indicator that shows if connext wallets were initialized.
```

Comment:
"Wallets" sounds a bit off here since there is really only one connext wallet, I'd just say "Whether the connext wallet was initialized." and have a similar comment for the restored node response below.